docs/UsersGuide.asciidoc

```diff
@@ -550,6 +550,19 @@ the top of the group overview page. However, it only works as operator
 or admin.
 
 
+=== Dark mode
+
+A dark mode theme can be enabled via "Appearance" settings for all logged in users. It can either be forced with
+the "dark mode" setting, or left to browser detection. Switching automatically between light and dark mode is natively
+supported by most modern browsers and can also be controlled manually via flags:
+
+ * On Firefox, go to `about:preferences#general` and search for "Website appearance".
+ * On Chrome, go to `chrome://flags/` and search for "Dark mode".
```

Comment:
I don't think setting chrome://flags will trigger the dark mode you're introducing here. At least on my system it seems to only enforce some Chromium-provided CSS overrides. So maybe it doesn't make sense to point it our here.

Reply:
Did you refresh the page after you changed that settings?

Reply:
I've tested this in the Chrome Inspector, it has an override feature for `prefers-color-scheme` where you can toggle back and forth. So i'm very sure that it works as it should.

Reply:
Ah, nice. Then it affects the media query despite also seemingly enforcing its own CSS rules on pages with no darkmode support at all. (Maybe it is also now already different since the time I tested it.)